cypress/integration/votepage.js

```diff
@@ -0,0 +1,34 @@
+describe("Load voting and render the expected page elements", () => {
+  it("Should confirm that true is equal to true", () => {
+    expect(true).to.equal(true);
+  });
+  it("Should be able to visit the page and render the header", () => {
+    cy.visit("http://localhost:3000")
+      .get("div.header")
+      .should("be.visible");
+  });
+  it("Should be able to visit the page and render the 'See Saved' button", () => {
+    cy.visit("http://localhost:3000")
+      .get(".saved-button")
+  });
+  it("Should be able to visit the page and render the book cover container", () => {
+    cy.visit("http://localhost:3000")
+        .get(".cover-img");
```

Comment:
You could a lot do one it block for all the .get()s since you are just checking they are on the page.  :-)